src/components/application_manager/test/commands/mobile/reset_global_properties_test.cc

```diff
@@ -77,6 +77,9 @@ typedef SharedPtr<ResetGlobalPropertiesResponse>
 namespace {
 const uint32_t kConnectionKey = 2u;
 const uint32_t kCorrelationId = 10u;
+const std::string kTimeOutDefaultValue = "time_out";
```

Comment:
@dcherniev please rename it to kTimeOutPromtDefaultValue
Because it is confusing that kTimeOutDefaultValue is string